frontend/src/metabase/lib/request.js

```diff
@@ -1,6 +1,4 @@
-import { AsyncApi } from "metabase/services";
 import _ from "underscore";
```

Comment:
why is this being removed?

Reply:
It's only used in xrays and my understanding was we'll eventually move to a more comprehensive solution rather than what was done for xrays.